server/src/test/java/uk/ac/cam/cl/echo/extrusionfinder/server/parts/MatchedPartTester.java

```diff
@@ -16,15 +16,15 @@
      */
     @Test
     public void testEquals() {
-        MatchedPart a = new MatchedPart(new Part("a", "b", "c", "d"), 0.02);
+        MatchedPart a = new MatchedPart(new Part("a", "b", "c", "d", "e"), 0.02);
 
-        assertTrue(new MatchedPart(new Part("a", "b", "c", "d"), 0.02).equals(a));
+        assertTrue(new MatchedPart(new Part("a", "b", "c", "d", "e"), 0.02).equals(a));
 
-        assertTrue(!new MatchedPart(new Part("A", "b", "c", "d"), 0.02).equals(a));
-        assertTrue(!new MatchedPart(new Part("b", "B", "c", "d"), 0.02).equals(a));
-        assertTrue(!new MatchedPart(new Part("b", "b", "C", "d"), 0.02).equals(a));
-        assertTrue(!new MatchedPart(new Part("b", "b", "c", "D"), 0.02).equals(a));
-        assertTrue(!new MatchedPart(new Part("b", "b", "c", "d"), 0.03).equals(a));
+        assertTrue(!new MatchedPart(new Part("A", "b", "c", "d", "e"), 0.02).equals(a));
+        assertTrue(!new MatchedPart(new Part("b", "B", "c", "d", "e"), 0.02).equals(a));
+        assertTrue(!new MatchedPart(new Part("b", "b", "C", "d", "e"), 0.02).equals(a));
+        assertTrue(!new MatchedPart(new Part("b", "b", "c", "D", "e"), 0.02).equals(a));
```

Comment:
There is one in the PartTester. Actually I think in MatchedPartTester, we probably only need one test where the Parts are equal and one where they are not. The details of how they differ are left to the PartTester.